File: app/src/main/java/social/entourage/android/map/tour/TourInformationModule.java
package social.entourage.android.map.tour;

import dagger.Module;
import dagger.Provides;

/**
 * Module related to TourInformationFragment
 * @see TourInformationFragment
 */
@Module
public class TourInformationModule {
    private final TourInformationFragment fragment;

    public TourInformationModule(final TourInformationFragment fragment) {
        this.fragment = fragment;
    }

    @Provides
    public TourInformationFragment providesTourInformationFragment() {
        return fragment;
    }
}

File: app/src/main/java/social/entourage/android/tour/TourInformationModule.kt
package social.entourage.android.tour

import dagger.Module
import dagger.Provides

/**
 * Module related to TourInformationFragment
 * @see TourInformationFragment
 */
@Module
class TourInformationModule(private val fragment: TourInformationFragment) {
    @Provides
    fun providesTourInformationFragment(): TourInformationFragment {
        return fragment
    }

}